docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,19 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 21 December 2022
+Frontend version: 1.0.0-20221221.0307  | Backend version: 1.40.4-221220.1019
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Manage application role assignments using TechBiz portal</summary>
+
+From 4 January 2023, if agencies can subscribe to an SGTS service (known as Tenant in TechPass portal) via the TechBiz portal, agencies need to use [TechBiz portal](https://portal.stg.techbiz.suite.gov.sg/) to manage their application role assignments.
+
+Currently, agencies manage application role assignments via TechPass portal or Automation API.
```

Comment:
currently agencies do not manage this in TP, it's the tenant themselves who manage this in TP. once the tenants are on TB, the agencies can manage it on their own on TB

Reply:
done.